### ISSUE TITLE
Improve ux/a11y of field validation errors

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#1718]
Example: omit a required field from the registrar signup form. Page re-renders with focus at the top of the page, and no clear evidence of what happened, unless you scroll down and inspect the form fields, where the error messages can be pretty subtle.

To fix: add js (likely to all pages, or all form pages) that automatically sets focus on some special `#alert-notification` type element, which should be displayed near the top of the page, and should if possible contain links to the fields with errors.
